appsettings/appsettings_test.go

```diff
@@ -20,7 +20,7 @@ const appSettingsFixture string = `
     "routable": true,
     "whitelist": ["1.2.3.4", "0.0.0.0/0"],
     "autoscale": {"cmd": {"min": 3, "max": 8, "cpu_percent": 40}},
-    "label": {"git_repo": "https://github.com/deisthree/controller-sdk-go", "team" : "deis"},
+    "label": {"git_repo": "https://github.com/teamhephy/controller-sdk-go", "team" : "deis"},
```

Comment:
team: "deis"

Not sure what this does...

Reply:
It's just a field on the labels of the appsettings route on controller API. Test passes so we can keep this label the same for now. The controller route it is supposed to mimic is `v2/apps/example-go/settings`